scripts/pre_commit_linter.py

```diff
@@ -57,6 +58,10 @@
     '--path',
     help='path to the directory with files to be linted',
     action='store')
+_PARSER.add_argument(
+    '--silent',
```

Comment:
Ah, sorry, ignore that. I see that the issue is that the linter is examining all of third_party as well. Would it be possible, when filtering for all files, to exclude /third_party and core/templates/prod?